DSNP-Wallet/Source/Modules/TestViewController.swift

```diff
@@ -65,64 +71,89 @@ extension TestViewController {
     }
     
     @objc func tappedBtn(selector: UIButton?) {
-        let completion: ExtrinsicSubmitClosure = { result in
-            var text = ""
-            
-            switch result {
-            case .success(let result):
-                text = result
-                
-            case .failure(let error):
-                text = "\(error)"
-            }
-            
-            let alert = UIAlertController(title: "Result", message: text, preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: "Ok", style: .cancel, handler: nil))
-            self.present(alert, animated: true)
-        }
-        
         let primaryMnemonicInput = primaryTextField.text?.isEmpty ?? true ? "quote grocery buzz staff merit patch outdoor depth eight raw rubber once" : primaryTextField.text ?? ""
         let primaryUser = User(mnemonic: primaryMnemonicInput)
         
         let secondaryMnemonicInput = secondaryTextField.text?.isEmpty ?? true ? "wink settle steak second tuition whale question must honey fossil spider melt" : secondaryTextField.text ?? ""
         let secondaryUser = User(mnemonic: secondaryMnemonicInput)
         
+        //Closures
+        let subscriptionIdClosure = getSubscriptionIdClosure()
```

Comment:
I want to make sure I understand this from a high level. We have a subscription closure to respond to changes in the tx status?

Reply:
I actually need to remove this